src/components/Book.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] it would be better to remove the eslint-disable because it ensures clean and professional code